task/source-build/0.3/source-build.yaml

```diff
@@ -46,6 +54,13 @@ spec:
   volumes:
     - name: source-build-work-place
       emptyDir: {}
+    - name: trusted-ca
+      configMap:
+        name: $(params.caTrustConfigMapName)
+        items:
+          - key: $(params.caTrustConfigMapKey)
+            path: ca-bundle.crt
+        optional: true
```

Comment:
Now that we've declared the volume in the non oci-ta task, should we also mount it? Probably in the `stepTemplate` so that it applies to all steps